src/clients/security_auto.rb

```diff
@@ -56,30 +56,30 @@ def main
       @param = {}
 
       # Check arguments
-      if Ops.greater_than(Builtins.size(WFM.Args), 0) &&
-          Ops.is_string?(WFM.Args(0))
-        @func = Convert.to_string(WFM.Args(0))
-        if Ops.greater_than(Builtins.size(WFM.Args), 1) &&
-            Ops.is_map?(WFM.Args(1))
-          @param = Convert.to_map(WFM.Args(1))
+      if !Yast::WFM.Args.empty?
+        @func = Yast::WFM.Args[0]
+        if Yast::WFM.Args.length > 1 && Yast::WFM.Args[1].is_a?(Hash)
```

Comment:
in fact second option is enough, because if length is 1 or less, then `Yast::WFM.Args[1]` returns nil and `nil.is_a?(Hash)` returns false